README.md

```diff
@@ -50,13 +50,42 @@ cd sc-machine/scripts
 ./run_sc_server.sh
 ```
 
-For running sc-server in authorization server mode, you need to generate a key pair. This can be done with next commands:
+For running sc-server in authorization server mode, you need a key pair. Current version of http service supports autogeneration of these keys. However you can done this manually with next commands:
```

Comment:
After installation on pure system keys still not generated. Generation by hands works well
![image](https://user-images.githubusercontent.com/34061914/159479601-0a6aee40-75d0-4b25-9c02-497c5b76f35c.png)